docs/getting-started/install-configure.md

```diff
@@ -243,7 +243,7 @@ provider that can satisfy a `PostgreSQLInstance`. Let's get started!
 > section.
 
 ```console
-kubectl crossplane install configuration registry.upbound.io/xp/getting-started-with-aws:latest
+kubectl crossplane install configuration registry.upbound.io/xp/getting-started-with-awsv1.9.1
```

Comment:
looks like we've lost the `:` in these latest -> 1.9.1 versions/tags - that's not intentional, right?

Reply:
It isn't! Thanks for catching!